### PR TITLE
Add missing documentation for RandomNumberGenerator types

### PR DESCRIPTION
**Pull request by bartonjs**
cc @carlossanlop

### CODE REVIEW

File: xml/System.Security.Cryptography/RNGCryptoServiceProvider.xml [hunk 1]
```diff
@@ -440,11 +440,14 @@
...
-        <param name="offset">To be added.</param>
-        <param name="count">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="data">The array to fill with cryptographically strong random bytes.</param>
+        <param name="offset">The index of the array to start the fill operation.</param>
+        <param name="count">The number of bytes to fill.</param>
+        <summary>Fills the specified byte array with a cryptographically strong random sequence of values.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException"><paramref name="data" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ArgumentOutOfRangeException"><paramref name="offset" /> or <paramref name="count" /> is less than 0</exception>
```
Comment (carlossanlop) [inline xml/System.Security.Cryptography/RNGCryptoServiceProvider.xml]: Missing dot:
```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException"><paramref name="offset" /> or <paramref name="count" /> is less than 0.</exception>
```

File: xml/System.Security.Cryptography/RNGCryptoServiceProvider.xml [hunk 2]
```diff
@@ -440,11 +440,14 @@
...
       </Parameters>
       <Docs>
-        <param name="data">To be added.</param>
-        <param name="offset">To be added.</param>
-        <param name="count">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="data">The array to fill with cryptographically strong random bytes.</param>
+        <param name="offset">The index of the array to start the fill operation.</param>
+        <param name="count">The number of bytes to fill.</param>
+        <summary>Fills the specified byte array with a cryptographically strong random sequence of values.</summary>
```
Comment (rpetrusha) [inline xml/System.Security.Cryptography/RNGCryptoServiceProvider.xml]:
```suggestion
        <summary>Fills the specified byte array with a cryptographically strong random sequence of values starting at a specified index for a specified number of bytes.</summary>
```
To more clearly disambiguate this overload from `GetBytes(Byte[])`.

File: xml/System.Security.Cryptography/RNGCryptoServiceProvider.xml [hunk 3]
```diff
@@ -534,10 +537,10 @@
         <Parameter Name="data" Type="System.Span&lt;System.Byte&gt;" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="data">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="data">The span to fill with cryptographically strong random nonzero bytes.</param>
+        <summary>Fills an array of bytes with a cryptographically strong random sequence of nonzero values.</summary>
```
Comment (rpetrusha) [inline xml/System.Security.Cryptography/RNGCryptoServiceProvider.xml]:
```suggestion
        <summary>Fills a byte span with a cryptographically strong random sequence of nonzero values.</summary>
```

File: xml/System.Security.Cryptography/RandomNumberGenerator.xml [hunk 4]
```diff
@@ -512,10 +514,22 @@ To create a random number generator, call the @System.Security.Cryptography.Rand
...
+## Remarks  
+  
+ The upper bound is exclusive to enable passing in a length value from an array, span, or list.
+ Because it is an exclusive upper bound this method can never generate <xref:System.Int32.MaxValue?displayProperty=nameWithType>.
+
+ This method uses a discard-and-retry strategy to avoid the low value bias that a simple modular arithmetic operation would produce.
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentOutOfRangeException">The <paramref name="toExclusive"/> parameter is 0 or less.</exception>
```
Comment (rpetrusha) [inline xml/System.Security.Cryptography/RandomNumberGenerator.xml]:
```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException">The <paramref name="toExclusive"/> parameter is less than or equal to 0.</exception>
```

File: xml/System.Security.Cryptography/RandomNumberGenerator.xml [hunk 5]
```diff
@@ -632,10 +660,10 @@ To create a random number generator, call the @System.Security.Cryptography.Rand
         <Parameter Name="data" Type="System.Span&lt;System.Byte&gt;" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="data">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="data">The span to fill with cryptographically strong random nonzero bytes.</param>
+        <summary>Fills an array of bytes with a cryptographically strong random sequence of nonzero values.</summary>
```
Comment (rpetrusha) [inline xml/System.Security.Cryptography/RandomNumberGenerator.xml]:
```suggestion
        <summary>Fills a byte span with a cryptographically strong random sequence of nonzero values.</summary>
```

**Comment by carlossanlop** [comment]
@rpetrusha @mairaw the suggestions where applied and the build passed. Is this good to merge?

**Comment by rpetrusha** [comment]
Thanks for the additional changes. I'll merge now, @carlossanlop and @bartonjs.

**Pull request merged**